Update README.md (#546)
carlosperate/awesome-microbit: README.md

- [Espruino JavaScript](https://www.espruino.com/MicroBit) - Contains an in-browser Blocky editor that can program micro:bit wirelessly. Also supports Bluetooth LE functionality.
- [Tinkercad Circuits](https://www.tinkercad.com/learn/circuits) - Create, code, and simulate electronic designs using common components with the micro:bit or Arduino, [related article](https://blog.tinkercad.com/explore-microbit-with-tinkercad).
- [OpenBlock](https://openblockcc.github.io/en/) - Block programming with hardware device support, including micro:bit, with code generation, compilation, flashing, and serial connection.

> - [MicroCode](https://microsoft.github.io/microcode/) - MicroCode is a icon-based (minimal text), computer friendly editor for the micro:bit V2 - can also program on an Arcade Shield with the correct hex code. It is suitable for younger learners and users with variable accessibility needs. (https://microsoft.github.io/microcode/docs/manual).

Reviewer: ```suggestion
- [MicroCode](https://microsoft.github.io/microcode/) - MicroCode is a icon-based (minimal text), editor for the micro:bit V2, which can also be used to write programmes directly on the micro:bit with an Arcade Shield. It is suitable for younger learners and users with variable accessibility needs.
```
